recipes/oatpp/all/conanfile.py

```diff
@@ -61,7 +61,7 @@ def validate(self):
 
     def build_requirements(self):
         if Version(self._version) >= "1.3.0":
-            self.tool_requires("cmake/[>=3.20 <4]")
+            self.tool_requires("cmake/[>=3.20]")
 
```

Comment:
This `build_requirements` scope is not needed as no version in CCI really requires CMake >=3.20, but CMake 3.1. And, it's expected to have CMake 3.15 as the minimum version installed for Conan 2.x. Could you please remove it?